src/scout_apm/core/cli/core_agent_manager.py

```diff
@@ -1,39 +1,40 @@
-#!/usr/bin/env python
```

Comment:
Isn't this needed if a user wants to execute this file from a shell?

Reply:
Yes, but executing this file from a shell doesn't do anything (as it currently stands).

Reply:
Ah, right. I forgot the `entry_points` spec handles setting up CLI scripts.